ERP/resources/views/components/navbar.blade.php

```diff
@@ -66,6 +57,17 @@
                     <a class="nav-link active" href="/machine-status">Machine Status</a>
                 </li>
                 @endif
+                
+                @if(Auth::user()->user_type==6)
+                <!--Redirects to Accountant page if the user has permission-->
+                <li class="nav-item">
+                    <a class="nav-link active" href="{{route('accountant')}}">Accountant</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link active" href="{{route('sales')}}">Sales</a>
+                </li>
```

Comment:
Accountant does not have access to Sales, it is the sales person that has access to Sales page.
The endif should end at line 66